### ISSUE TITLE
Skimming through with <space>

### ISSUE THREAD

**Issue by hfiguiere** [jangernert/FeedReader#867]
One thing I miss is when reading an article, is being able to use the space bar to go forward (and shift-space for backward). U/I for scrolling doesn't make it. Space would scroll a page and at the end to the next unread article in the feed.

**Comment by brendanlong** [comment]
I think you might be able to change the hotkeys in dconf, although I don't think we have a UI for it right now.

**Comment by hfiguiere** [comment]
Yes and no. This is a combination of "K", Ctrl-K and page down. Not just binding one key to the other.

Examples of this pattern: Thunderbird, Liferea.
